shared/featureconfig/config.go

```diff
@@ -123,41 +123,51 @@ func InitWithReset(c *Flags) func() {
 	return resetFunc
 }
 
-// ConfigureBeaconChain sets the global config based
-// on what flags are enabled for the beacon-chain client.
-func ConfigureBeaconChain(ctx *cli.Context) {
-	// Using Medalla as the default configuration for now.
-	params.UseMedallaConfig()
-
-	complainOnDeprecatedFlags(ctx)
-	cfg := &Flags{}
-	if ctx.Bool(devModeFlag.Name) {
-		enableDevModeFlags(ctx)
-	}
+// configureTestnet sets the config according to specified testnet flag
+func configureTestnet(ctx *cli.Context, cfg *Flags) {
 	if ctx.Bool(AltonaTestnet.Name) {
```

Comment:
I think using switch/case statements will be better here

Reply:
I am not sure you'll be able to use a switch with these

Reply:
yeah i'm not sure too, it's possible but it's boolean flags it will be complicated

Reply:
you can definitely use switch statements here 
```go
switch {
case condition_1 :
....
case condition_2:
}
```
Realised my review might have been too late here